### ISSUE TITLE
What is your performance?

### ISSUE THREAD

**Issue by John1231983** [qq456cvb/mask-rcnn#1]
Thanks for your project!. Could you provide your current performance on coco? I am using maskrcnn on keras and it achieve around 0.29 mAP on segmentation task. (paper is 0.34)

**Comment by qq456cvb** [comment]
I have not trained it yet since it may take a long time without powerful GPU clusters. Do you train the model from scratch or fine-tuning some pretrained model?

**Comment by John1231983** [comment]
I want to train the model from pre-train model likes resnet101. I found your repo, but I don't know it is ready to run or not. Is it ready?

**Comment by qq456cvb** [comment]
It is not ready yet, since some mask data preprocessing and Tensorflow higher API integration has to be done. Besides, I am using Resnet50 model.
